libr/main/radiff2.c

```diff
@@ -696,6 +719,89 @@ static ut8 *get_strings(RCore *c, int *len) {
 	return buf;
 }
 
+static char *get_graph_commands (RCore *c, ut64 off) {
+        char *retstr;
+        const char *static_str = NULL;
+        int tmp_html = r_cons_singleton ()->is_html;
+        r_cons_singleton ()->is_html = 0;
```

Comment:
use false, not 0, this is bool, or it shuold...

Reply:
It is actually an int sir, but I changed it to false